### ISSUE TITLE
After uninstall desktop unstable and without icons

### ISSUE THREAD

**Issue by falko** [iabem97/komorebi#106]
After uninstalling Komorebi:
When I minimize the last window, I see the Ubuntu 16.04 default wallpaper or sometimes a black screen and no desktop icons and sometimes the X server freezes with 100% CPU load. I can't change the wallpaper in Ubuntu.

**Comment by kpuljek** [comment]
Yup, same here. Removed it completely, purged all settings, it's still broken

**Comment by sriharshakappala** [comment]
@kpuljek You are a life saver!

**Comment by falko** [comment]
Thx @kpuljek! That fixed it for me too.

Nevertheless, this should be automated by the uninstall scripts of Komorebi

**Comment by sweetquiet** [comment]
#!/bin/bash
echo "[INFO] Ignore any errors"
echo "[*] Resetting desktop configuration"
gsettings set org.gnome.desktop.background show-desktop-icons true || true
gsettings set org.nemo.desktop show-desktop-icons true || true

**Comment by BadWolf47** [comment]
> Yup, same here. Removed it completely, purged all settings, it's still broken.
> 
> Edit: Got the desktop back by running this in the terminal: gsettings set org.gnome.desktop.background show-desktop-icons true

I have to run this command every time i boot my PC because the icons don't show up and right-click doesn't work.
